src/core/resources/resource.js

```diff
@@ -95,6 +95,11 @@ Resource.prototype.redisIn = function (data) {
   })
 
   this.emit('message:outgoing', data)
+
+  if (!Object.keys(this.subscribers).length) {
```

Comment:
I know there's probably some places in the radar codebase that aren't consistent, but for new code prefer explicit comparisons over boolean coercion: `if (Object.keys(this.subscribers).length === 0) {...`